### DISCUSSION TITLE
Dynamic wildcard subdomains

### DISCUSSION THREAD

**Discussion by charbs-io** [vercel/community#3999] [Help] [answered]
### Summary

Hi, 

New to Vercel (coming from Netlify). 

I have been building a POC and hit a wall in Netlify hence trying Vercel out.

I have a somewhat unique (but not too crazy) usecase in that I need a page to be redirected to a subdomain and the user served the content there. It's a Nuxt 3 app.

example.com - served by /index.vue
example.com/store - served by /store/index.vue
example.com/store/:storeslug - served by /store/[slug].vue

That last URL needs to be redirected to :storeslug.example.com. I had that working on Netlify via a simple `_redirects` file which they support.

```
/store/:storeslug/*  https://:storeslug.example.com/:splat  301
```

Is this sort of thing not supported on Vercel? All docs i've read and seen point to... No?...

Any help/pointers would be really appreciated.

### Steps to Reproduce

(this is not a bug)

**Comment by amyegan** [comment, marked as answer]
Hey @charbs-io. This is possible, but I'm not sure of your exact setup or what you're comfortable using. There are a few options for handling the redirect and a couple of ways to set up the wildcard subdomain. So I'm just going to share a short list of links to a few pages that have the relevant info. 

- [Redirects](https://vercel.com/docs/edge-network/redirects#)
- [Introducing Wildcard Domains](https://vercel.com/blog/wildcard-domains)
- [Using wildcard domain](https://vercel.com/docs/projects/domains/add-a-domain#using-wildcard-domain)
- [Can I use wildcard domains without switching to Vercel Nameservers?](https://vercel.com/guides/wildcard-domain-without-vercel-nameservers)

Please let me know if this isn't enough information or if there is anything else you need 🙂

**Reply by Youhan** [reply]
I have a simillar issue I think.
I want this behaviour:

When you go to wildcard.domain.com, I want to setup a dynamic rewrite (not a redirect).

At the end, as a user, your browser is showing something.domain.com but the content is handled by domain.com/profile/billy. 

You can also go to domain.com/profile/billy to see the same content.

Any help?

**Reply by orpic** [reply]
did you figure this out ??

i have my dns records in aws route 53,
ideally i want a setup where wildcard subdomains point a single vercel deployement and in vercel i also want a project which handles wildcard subdomain to a deployment and do the same rewrite as you mentioned
